tests/test_bgp_af.rb

```diff
@@ -828,7 +1117,15 @@ def network_cmd(af, dbg)
                  "#{dbg} Test 3. Restore the removed networks")
 
     # Test: Change route-maps on existing networks
-    should = master.map { |network, rm| [network, "#{rm}_55"] }
+    if platform == :ios_xr
+      %w(rtmap1_55, rtmap2_55, rtmap3_55, rtmap5_55,
+         rtmap6_55, rtmap7_55).each do |policy|
+        config("route-policy #{policy}", 'end')
+      end
+      should = master.map { |network, rm| [network, rm.nil? ? nil : "#{rm}_55"] }
```

Comment:
I’ve only been able to test it on XR so far.  If I can get my hands on a Nexus I’ll test it.

> On 02 Dec 2015, at 17:33, Glenn Matthews <notifications@github.com> wrote:
> 
> In tests/test_bgp_af.rb <https://github.com/cisco/cisco-network-node-utils/pull/87#discussion_r46437835>:
> 
> > @@ -828,7 +1117,15 @@ def network_cmd(af, dbg)
> >                   "#{dbg} Test 3. Restore the removed networks")
> >  
> >      # Test: Change route-maps on existing networks
> > -    should = master.map { |network, rm| [network, "#{rm}_55"] }
> > +    if platform == :ios_xr
> > +      %w(rtmap1_55, rtmap2_55, rtmap3_55, rtmap5_55,
> > +         rtmap6_55, rtmap7_55).each do |policy|
> > +        config("route-policy #{policy}", 'end')
> > +      end
> > +      should = master.map { |network, rm| [network, rm.nil? ? nil : "#{rm}_55"] }
> OK, but seems like this would be an issue for NXOS too. Why make it XR-specific?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cisco/cisco-network-node-utils/pull/87/files#r46437835>.
>